docs/context/ref/ContactList.md

```diff
@@ -64,6 +64,7 @@ Other Types
 Intents
 - [StartChat](../../intents/ref/StartChat)
 - [StartCall](../../intents/ref/StartCall)
+- [CreateInteraction](../../intents/ref/CreateInteraction)
```

Comment:
alpha sort or rebase PR so this change isn't in here. If you've fixed in the other PR you can merge it into this to bring across - although we prefer clean PRs based off master (rather than branches used in other PRs)